src/algorithms/tracking/TrackSeeding.cc

```diff
@@ -86,10 +86,19 @@ std::vector<edm4eic::TrackParameters*> eicrecon::TrackSeeding::makeTrackParams(S
       float R = std::get<0>(RX0Y0);
       float X0 = std::get<1>(RX0Y0);
       float Y0 = std::get<2>(RX0Y0);
-      if (R > std::sqrt(std::cbrt(std::numeric_limits<float>::max()))) {
-        // avoid future float overflow for hits on a line
+      if (!(std::isfinite(R) &&
+	std::isfinite(std::abs(X0)) &&
+	std::isfinite(std::abs(Y0)))) {
+        // avoid float overflow for hits on a line
         continue;
       }
+      if ( std::hypot(X0,Y0) < std::numeric_limits<double>::epsilon() ||
```

Comment:
Would like to point out that `X0` and `Y0` are `float`s. Perhaps
```suggestion
      if ( std::hypot(X0,Y0) < std::numeric_limits<decltype(std::hypot(X0,Y0))>::epsilon() ||
```